.travis.yml

```diff
@@ -6,8 +6,6 @@ cache: bundler
 
 rvm:
   - ruby-head
-  - 2.1.2
-  - 2.0.0
-  - 1.9.3
-  - jruby-19mode
-  - rbx-2.2.10
+  - 2.1.9
```

Comment:
Use 2.3.1

Feel free to remove jruby or newer versions of rbx. If you want to leave them, feel free to ignore failures on rbx-3.0, it's always a moving target.